test/e2e_node/runtime_conformance_test.go

```diff
@@ -85,10 +85,9 @@ var _ = Describe("Container runtime Conformance Test", func() {
 						},
 					},
 				}
-				testCount := int32(3)
 				testStatuses := []testStatus{
-					{"terminate-cmd-rpa", api.RestartPolicyAlways, api.PodRunning, ContainerStateWaiting | ContainerStateRunning | ContainerStateTerminated, ">", testCount, false},
-					{"terminate-cmd-rpof", api.RestartPolicyOnFailure, api.PodSucceeded, ContainerStateTerminated, "==", testCount, false},
+					{"terminate-cmd-rpa", api.RestartPolicyAlways, api.PodRunning, ContainerStateWaiting | ContainerStateRunning | ContainerStateTerminated, "==", 2, true},
```

Comment:
Why do we still need `ContainerStateWaiting | ContainerStateRunning | ContainerStateTerminated`?
The container must be running, right?